ansible_rulebook/action/debug.py

```diff
@@ -60,7 +64,9 @@ async def __call__(self):
                 value = dpath.get(
                     self.helper.control.variables, key, separator="."
                 )
-                self.display.banner("debug", f"{key}: {value}")
+                self.display.banner(
```

Comment:
@zkayyali812 If we did this once in the rule_set_runner here https://github.com/ansible/ansible-rulebook/blob/8734287ffdd88d41e69c6db75ef4a3e556ea1770/ansible_rulebook/rule_set_runner.py#L410 if the action in ['debug','print_event'] we can swap the variables_copy with the masked copy and everything else should just use the masked variables.

Reply:
@mkanoor This does not work in my testing. When I tested with this the substitution does not happen unless manually specified as done in this PR.